host_vars/qld-pulsar-himem-0.yml

```diff
@@ -18,7 +18,8 @@ pulsar_rabbit_username: "galaxy_qld_hm0"
 pulsar_rabbit_vhost: "/pulsar/galaxy_qld_hm0"
 
 attached_volumes:
-  - device: /dev/vdb1
+  - device: /dev/vdb
+    partition: 1
```

Comment:
this is an optional var and we can use it on all new VMs and this role should still work with old ones..